packages/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDelegate.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <UMCore/UMDefines.h>
+
+@class EXSessionTaskDelegate;
+
+@interface EXSessionTaskDelegate : NSObject <NSURLSessionTaskDelegate>
```

Comment:
Do you think it would make sense to change the architecture so that `EXFileSystem` is only responsible for creating and delegating actions to tasks and let the tasks do the actual work? I'm afraid firing off tasks and forgetting about the delegates (i.e. not clearing them when the bridge becomes invalid) may cause errors when resolving/rejecting promises when the app is refreshing.